doc/cli/vast-import.md

```diff
@@ -82,6 +82,17 @@ size.
 
 #### `vast.import.batch-timeout`
 
-Sets a timeout for forwarding buffered table slices to the importer. If the
-timeout fires before a table slice reaches `vast.import.batch-size`, then the
-table slice will contain fewer events but ship immediately.
+Sets a timeout for forwarding buffered table slices to the importer.
+
+The `vast.import.batch-timeout` option controls the maximum buffering period
+until table slices are forwarded to the node. The default batch timeout is one
+second.
+
+#### `vast.import.read-timeout`
+
+Sets a timeout for reading from input sources.
+
+The `vast.import.read-timeout` option controls the time that a call to read data
+from the input will block. The process yields and tries again at a later time if
+no data is received for the set value. The default read timeout is 20
+milliseconds.
```

Comment:
The first sentence does not read very fluently.

Reply:
Updated to "determines how long a call to read data from the input will block".